pkg/osutil/osutil.go

```diff
@@ -333,10 +333,13 @@ func Abs(path string) string {
 	if wd1, err := os.Getwd(); err == nil && wd1 != wd {
 		panic(fmt.Sprintf("wd changed: %q -> %q", wd, wd1))
 	}
-	if path == "" || filepath.IsAbs(path) {
+	if path == "" {
```

Comment:
Abs("") case is not documented.
Why don't we filepath.Clean() in this case?

Reply:
Got it "Clean returns the shortest path name equivalent to path".

Reply:
https://go.dev/play/p/o5kdt5mjjt7